experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -251,7 +251,7 @@ export class NodeSDK {
       this._resourceDetectors = [];
     } else if (configuration.resourceDetectors != null) {
       this._resourceDetectors = configuration.resourceDetectors;
-    } else if (process.env.OTEL_NODE_RESOURCE_DETECTORS != null) {
+    } else if (getBooleanFromEnv('OTEL_NODE_RESOURCE_DETECTORS')) {
```

Comment:
```suggestion
    } else if (getStringFromEnv('OTEL_NODE_RESOURCE_DETECTORS')) {
```

This envvar is not a boolean. Using `getBooleanFromEnv` would accidentally not use the envvar if `OTEL_NODE_RESOURCE_DETECTORS=false` was used.

Using `getStringFromEnv` is a *slight* behaviour change because it will now consider the envvar to be empty if it is just a string of whitespace. However, I think that is fine.

Reply:
Ah, good catch. Sorry about that! Fixed.